sandy/groupr.py

```diff
@@ -147,11 +166,13 @@ def get_xs(self, **kwargs):
                                           listmt=listmt_,
                                           listmat=listmat_).data:
             mf3 = sandy.groupr.read_mf3(self, mat, mt)
-            xs = np.array([x["DATA"][1].tolist() for x in mf3["GROUPS"]]).T
+            lowest_range = mf3["GROUPS"][0]["IG"] - 1
+            xs = np.array([x["DATA"][1].tolist() for x in mf3["GROUPS"]])
+            xs = np.insert(xs, [0]*lowest_range, 0) if lowest_range != 0 else xs
```

Comment:
@AitorBengoechea . This is ok, but pandas bfill could be better, i guess...

Reply:
As it was: xs in the array of the values of the xs and columns is the `pd.MultiIndex` and index is the `pd.Index`. For me is more natural continue working with np in the xs variable and them in then combine all in the `data.append`